oss.gg/3_follow_the_unkey_x_account.md

```
@@ -25,4 +25,8 @@ Your turn 👇
 » 07-October-2024 by [Harsh Bhat](@HarshBhatX)
 » Link to Account: https://x.com/HarshBhatX
 
+» 11-October-2024 by [Chirag Arora](@Chirag8023)
+» Link to Account: https://x.com/chirag8023
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)